schema/proto3/goalstate.proto

```diff
@@ -24,10 +24,12 @@ import "vpc.proto";
 import "subnet.proto";
 import "port.proto";
 import "securitygroup.proto";
+import "dhcp.proto";
 
 message GoalState {
```

Comment:
We need to discuss the naming of GoalState.

Reply:
I got the idea now and I think we can close this topic.